testConnector/src/test/java/io/vantiq/extsrc/testConnector/TestTestConnectorCore.java

```diff
@@ -35,13 +32,15 @@
 
     private static String environmentVariable;
     private static String filename;
+    private static String fileContents;
 
     private static final String NONEXISTENT_ENV_VAR = "anEnvironmentVariableThatIsUnlikelyToExist";
 
     @BeforeClass
     public static void checkFilesAndEnvVars() {
         environmentVariable = System.getProperty("TestConnectorEnvVarName", null);
         filename = System.getProperty("TestConnectorFilename", null);
+        fileContents = System.getProperty("TestConnectorFileContents", null);
```

Comment:
just FYI, I think you can leave out the 2nd parameter & it'll default to null.  Usually you need that only to provide a non-null default.

Reply:
Fixed